ts/features/payments/details/components/PaymentsMethodDetailsBaseScreenComponent.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  ContentWrapper,
+  IOColors,
+  IOSpacingScale,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import { ScrollView } from "react-native-gesture-handler";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import FocusAwareStatusBar from "../../../../components/ui/FocusAwareStatusBar";
+import { useIOSelector } from "../../../../store/hooks";
+import { isDesignSystemEnabledSelector } from "../../../../store/reducers/persistedPreferences";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+import {
+  PaymentCard,
+  PaymentCardComponentProps
+} from "../../common/components/PaymentCard";
+
+type Props = {
+  card: PaymentCardComponentProps;
+  headerTitle?: string;
+};
+
+/**
+ * Base layout for payment methods screen & legacy delete handling
+ */
+const PaymentsMethodDetailsBaseScreenComponent = ({
+  card,
+  headerTitle = "",
+  children
+}: React.PropsWithChildren<Props>) => {
+  const isDSenabled = useIOSelector(isDesignSystemEnabledSelector);
+  const blueHeaderColor = isDSenabled ? IOColors["blueIO-600"] : IOColors.blue;
+
+  return (
+    <BaseScreenComponent
```

Comment:
the `BaseScreenComponent` has `NativeBase` as a  dependency, are we sure that we want to keep using it, compared to something like `HeaderSecondLevel`?